modules/serverless-installing-cli-macos.adoc

```diff
@@ -3,16 +3,16 @@
 // * serverless/cli_tools/installing-kn.adoc
 
 :_content-type: PROCEDURE
-[id="installing-cli-macosx_{context}"]
+[id="serverless-installing-cli-macos_{context}"]
 = Installing the Knative CLI for macOS
 
-The `kn` CLI for macOS is provided as a `tar.gz` archive.
+If you are using macOS, you can install the Knative (`kn`) CLI as a binary file. To do this, you must download and unpack a `tar.gz` archive and add the binary to a directory on your `PATH`.
 
 // no prereqs?
 
 .Procedure
 
-. Download the link:https://mirror.openshift.com/pub/openshift-v4/clients/serverless/latest/kn-macos-amd64.tar.gz[`kn` CLI].
+. Download the link:https://mirror.openshift.com/pub/openshift-v4/clients/serverless/latest/kn-macos-amd64.tar.gz[Knative (`kn`) CLI `tar.gz` archive].
 
 . Unpack and unzip the archive.
```

Comment:
According to the IBMSG: Do not use "unzip" to refer to extracting or decompressing compressed data. Use "extract" or "decompress".